bokehjs/gulp/tasks/develop.coffee

```diff
@@ -8,8 +8,7 @@ subTasks = ["develop:install-js"]
 gulp.task "develop", subTasks, ->
 
 gulp.task "develop:watch", ["develop"], ->
-  gulp.watch "#{paths.buildDir.js}/*.js", ["develop:install-js"]
-  gulp.watch "#{paths.buildDir.css}/*css", ["develop:install-js"]
+  gulp.watch "#{paths.coffee.watchSources}", ["develop:install-js"]
```

Comment:
Well, my original idea might be outdated. :-)

I originally thought I needed to do the `python setup.py develop --install_js` to get it in a working state.  In that case, the idea was that you'd cd into `bokehjs` and run `gulp develop`.  At that point, everything would built, then installed.  I think you've fixed the root issue with the various new code you've added so we might be able to nix it (yay deleting code!).